ludicrousdb/includes/class-ludicrousdb.php

```diff
@@ -340,7 +340,7 @@ public function add_database( array $db = array() ) {
 		);
 
 		// Merge using defaults
-		$db      = wp_parse_args( $db, $database_defaults );
+		$db      = array_merge( $db, $database_defaults );
```

Comment:
The defaults will overwrite the values in `$db` this way round

Reply:
🤦